sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosAsyncContainer.java

```diff
@@ -758,6 +758,14 @@ <T> CosmosPagedFlux<T> queryChangeFeedInternal(
             queryChangeFeedInternalFunc(cosmosChangeFeedRequestOptions, classType));
     }
 
+    String getLinkWithoutTrailingSlash() {
```

Comment:
Probably can use Utils.trimBeginningAndEndingSlashes

Reply:
Probably - but not worth risking any regression after ADF tested this. Won't fix.